lib/main.coffee

```diff
@@ -22,6 +22,10 @@ module.exports =
       type: 'boolean'
       default: true
       description: 'When listing directory items, list subdirectories before listing files.'
+    posixSort:
```

Comment:
I think it would make sense to change this to `alphanumericSort` with a default of `true`. Thus in essence flipping the current logic.

I'm pretty sure there's nothing in the POSIX standard relating to sort order of directory listings, so calling it `posixSort` is a bit of a misnomer either way.